cmd/mc-router/main.go

```diff
@@ -43,6 +43,8 @@ type Config struct {
 	MetricsBackend       string   `default:"discard" usage:"Backend to use for metrics exposure/publishing: discard,expvar,influxdb"`
 	UseProxyProtocol     bool     `default:"false" usage:"Send PROXY protocol to backend servers"`
 	MetricsBackendConfig MetricsBackendConfig
+
+	CloudflareSRV bool `default:"false" usage:"Enable cleaning IP for cloudflare SRV as it breaks parsing"`
```

Comment:
...and let's call this something more generic like `SimplifySRV` since I have a feeling all minecraft client lookups of SRV will do the same.

Reply:
I would've done that, however I dont know how other services use their SRV records, so... I can only reliably account for one service's SRV records unless someone has logs for other services SRV records and what they claim the client is trying to connect to, 

I personally do not feel comfortable as of this moment claiming this is an "fix all" for all srv service providers.